pkg/crc/machine/start.go

```diff
@@ -167,6 +180,8 @@ func (client *client) Start(ctx context.Context, startConfig StartConfig) (*Star
 			if err != nil {
 				return nil, errors.Wrap(err, "Cannot create cluster configuration")
 			}
+
+			telemetry.SetContextProperty(ctx, telemetryStartType, alreadyRunning)
```

Comment:
This one feels odd, we will be sending `start-type: first-start`, `start-type: start`, or `start-type:start` followed by `start-type: already-running`. I'd expect only the latter to be sent in the last case, not to get 2 start-type events.

Reply:
already-running is the case where the duration is almost 0. 

This call overrides the value of previously set. Not ideal.. I will remove this.

Reply:
Yeah, took me a long time to realize `SetContextProperty` overwrites the previous value, so the behaviour is fine (only one `start-type` event sent). Once wrapped in a slightly higher level API, should be all fine!